packages/unit-testing/stubs/currentRecord/CurrentRecordInstance.js

```diff
@@ -0,0 +1,463 @@
+define(['./Sublist', './Field'], function (Sublist, Field) {
+    function CurrentRecord() {
+        /**
+         * return value of the field
+         * @param {Object} options
+         * @param {string} options.fieldId
+         * @return {(number|Date|string|Array|boolean)}
+         * @throws {SuiteScriptError} SSS_MISSING_REQD_ARGUMENT if fieldId is missing or undefined
+         * @throws {SuiteScriptError} SSS_INVALID_API_USAGE if invoked after using setText
```

Comment:
This second error doesn't appear in the help
https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=section_4637582256.html